addons/binding/org.openhab.binding.nest/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
All the channel types can't be found because of this "dscalarm" :-)

Reply:
o O

Reply:
The copy-paste-pattern well applied...

Reply:
That should be fixed